index.html

```diff
@@ -1,411 +1,381 @@
-                type="text"
-                placeholder="Fullname"
-                name="name"
-                maxlength="30"
-                required
-              />
-            </div>
-            <div class="form_field_item form_field_item2">
-              <input
-                type="email"
-                placeholder="Email address"
-                name="email"
-                required
-              />
-            </div>
-            <div class="form_field_item form_field_item3">
-              <textarea
-                maxlength="500"
-                name="description"
-                rows="5"
-                cols="30"
-                placeholder="Enter text here"
-                required
-              ></textarea>
-            </div>
-            <div class="form_field_item form_field_item4">
-              <button type="submit">
-                <span class="btn_text">Get in touch</span>
-              </button>
-            </div>
-          </form>
+        <div class="skill_item skill_item3">
+          <div class="skill_header">
+            <img src="img/skills/rectangle57.png" alt="icon" />
+            <h3>Skills</h3>
+          </div>
+          <div>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
         </div>
       </div>
-      <footer>
-        <div class="footer_social_media social_media">
-          <ul>
-            <li>
-              <a href="#">
-                <img
-                  class="img img1"
-                  src="img/footer/github.png"
-                  alt="GitHub"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img img2"
-                  src="img/footer/linkedin.png"
-                  alt="Linkedin"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img img3"
-                  src="img/footer/angelist_1.png"
-                  alt="Angelist"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img img4"
-                  src="img/footer/twitter.png"
-                  alt="Twitter"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img img5"
-                  src="img/footer/Vector.png"
-                  alt="Vector"
-              /></a>
-            </li>
-          </ul>
-        </div>
-      </footer>
-    </main>
-  </body>
-</html>
+    </section>
+    <div class="form">
+      <div class="form_item form_item1">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd
+          like to chat please get in touch.
+        </p>
```

Comment:
- [OPTIONAL] please consider using the same text font as the Figma design in this section to match the template as stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md).

Your design 👇 
![image](https://user-images.githubusercontent.com/78251927/206791636-66bcdadd-eb92-45b4-80cb-a0a5feb68e30.png)

Figma design 👇 
![image](https://user-images.githubusercontent.com/78251927/206791716-6dfa3ae1-e1e1-4c29-87ab-dec3d1d1e951.png)

Reply:
Hello @abdulhamiid  Thank you for the review. the font weight is the same, just that when applied it looks like this.